multisrc/overrides/mangathemesia/legacyscans/src/LegacyScans.kt

```diff
@@ -0,0 +1,7 @@
+package eu.kanade.tachiyomi.extension.fr.legacyscans
```

Comment:
```suggestion
package eu.kanade.tachiyomi.extension.fr.flamescansfr
```

Don't forget to rename the overrides folder from `legacyscans` back to `flamescansfr` as well.